public/app/plugins/panel/graph2/GraphLegendEditor.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import capitalize from 'lodash/capitalize';
+import without from 'lodash/without';
+import { StatID, LegendOptions, PanelOptionsGroup, Switch, Input } from '@grafana/ui';
+
+export interface GraphLegendEditorLegendOptions extends LegendOptions {
+  stats?: StatID[];
```

Comment:
stats should be `string[]`, StatID is just the list of standard stats, not the list of all available stats.